examples/webbilling-demo/src/tests/main.test.ts

```diff
@@ -8,19 +8,27 @@ const RC_PAYWALL_TEST_OFFERING_ID = "rc_paywalls_e2e_test_2";
 const RC_PAYWALL_TEST_OFFERING_ID_WITH_VARIABLES =
   "rc_paywalls_e2e_test_variables_2";
 
-const waitForCheckoutStartRequest = (page: Page) => {
-  return page.waitForRequest(
-    (request) =>
-      request.url().includes("checkout/start") && request.method() === "POST",
-  );
+const waitForCheckoutStartRequest = (
```

Comment:
Does it make sense moving this helper function to the bottom as the others?

Reply:
On it.